tests/test-notebooks/test_notebooks.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbconvert.preprocessors.execute import CellExecutionError
+
+import glob
+import traceback
+import unittest
+import sys
+import time
+import os
+
+
+class TestNotebooks(unittest.TestCase):
+  def _execNotebook(self, ep, notebook_filename, nbpath):
+    with open(notebook_filename) as f:
+      nb = nbformat.read(f, as_version=nbformat.current_nbformat)
+      try:
+        out = ep.preprocess(nb, {'metadata': {'path': nbpath}})
+      except CellExecutionError:
+        print('-' * 60)
+        traceback.print_exc(file=sys.stdout)
+        print('-' * 60)
+        self.assertTrue(False,
+                        'Error executing the notebook %s. See above for error.'
+                        % notebook_filename)
+
+  def test_all_notebooks(self):
+    ''' Test all notebooks except blacklist. '''
+    blacklist = ['gan.ipynb', 'iclr2017.ipynb']
```

Comment:
okay. i wasn't sure if this was a long-term notebook. if so, we may want to rename it to something a little bit more descriptive.